### PR TITLE
Described our release procedure as Makefile

### PR DESCRIPTION
**Pull request by yoichitgy**
This PR addresses Issue #476.

- Made `Makefile` to describe our release procedure.
- Add a section for maintainers to `CONTRIBUTING.md` to tell about the `Makefile`.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (yoichitgy) [inline CONTRIBUTING.md:54]: The link to Makefile will be available after this PR is merged.

File: Makefile
Comment (mpdifran) [inline Makefile:58]:
```suggestion
	$(error The version tag $(VERSION_TAG) does not match the version in Info.plist)
```

Comment (yoichitgy) [reply Makefile]: @mpdifran Thank you for finding and fixing the typo

**Comment by 1ucas** [comment]
I never released a Swinject version before. But it looks good to me!
Maybe we should take a look at an CD alternative, to ensure we don't forget to run any of the steps? 
Or even an hybrid solution, calling the Makefile during CD execution.
Example:
https://github.com/1ucas/Swinject/pull/3/files

**Review by mpdifran — APPROVED** [review]

**Comment by yoichitgy** [comment]
Thanks 1ucas! Yes, the release using CD is nice to maintain👍

**Comment by yoichitgy** [comment]
Thanks! I merged the PR! We can move to 1lucas's CD version later🚀

**Pull request merged**
